### ISSUE TITLE
AD logins do not work (workaround available)

### ISSUE THREAD

**Issue by fsamareanu** [nilsteampassnet/TeamPass#2377]
### Steps to reproduce
1. Fresh install of 2.1.27
2. Configure AD connectivity (works)
3. Upgrade to 2.1.27-17 (breaks AD account login)
4. Fresh install of 2.1.27-17 has identical behavior

### Expected behaviour
User should be able to log in

### Actual behaviour
User cannot log in

### Server configuration
**Operating system**:
RHEL 7.5

**Web server:**
Apache

**Database:**
mariadb 5.5

**PHP version:**
7.2

**Teampass version:** 
2.1.27-17

**Teampass configuration file:**

**Updated from an older Teampass or fresh install: both fresh install and upgrade behave the same
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

[tp_config.txt](https://github.com/nilsteampassnet/TeamPass/files/2293683/tp_config.txt)

### Client configuration
**Browser:**
Any (tried both IE, Chrome and Firefox LTS)

**Operating system:**
Windows 10 and 2012 R2

### Logs
[ldap_debug.txt](https://github.com/nilsteampassnet/TeamPass/files/2293688/ldap_debug.txt)
[duo_debug.txt](https://github.com/nilsteampassnet/TeamPass/files/2293689/duo_debug.txt)

#### Web server error log

No error in log file

Insert your webserver log here

Nothing of interest 
#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Nothing interesting
```

**Comment by fsamareanu** [comment]
A quick and dirty workaround is to comment the following lines in sources/identify.php starting with line 765:

 if (is_array($adldap->user()->passwordExpiry($auth_username)) === false) {
 echo '[{"value" : "user_not_exists '.$auth_username.'", "text":""}]';
 exit();
 }

**Comment by fsamareanu** [comment]
Disclaimer: I am not a php developer and I never touched php before.

I added a bit of debug code to the identify.php file (see attached txt file).
[identify.php.diff.txt](https://github.com/nilsteampassnet/TeamPass/files/2293891/identify.php.diff.txt)

This printed the following message to ldap debug log:

Expiry check output: Domain does not expire passwords

I've added the equality condition on that string and now AD Ldap users can log in successfully.

I'd like to thank @F0ob4r for pointing me to the right direction in #2368 .
